tests/spread/general/unicode-metadata/task.yaml

```diff
@@ -10,6 +10,10 @@ prepare: |
 
   base=$(get_base)
   sed -e "s/base: {{BASE}}/base: ${base}/g" expected_snap_tmpl.yaml > expected_snap.yaml
+  if [[ "$base" =~ "core22" ]]; then
```

Comment:
I couldn't figure out a better way to make this spread test work across core18, 20, and 22.

Reply:
this is fine for now, we will eventually hit a problem as we are not resetting the expected_snap.yaml, but I would not worry about that now.